ruby/tests/common_tests.rb

```diff
@@ -614,7 +614,6 @@ def test_oneof
     assert d2 == d
 
     encoded_field_a = proto_module::OneofMessage.encode(proto_module::OneofMessage.new(:a => "string"))
-    encoded_field_b = proto_module::OneofMessage.encode(proto_module::OneofMessage.new(:b => 1000))
```

Comment:
Do you need to remove the whole line?

Reply:
Should it be used some how later?

Reply:
I added `encoded_field_b` back in the latest commit so it is used in 2 places after, but I'm not sure what the original author's intent was.